src/pages/Contributors/index.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState, useEffect, useRef } from 'react'
+import { useStyle } from './styles.js';
+import axios from 'axios'
+import {Dropdown} from '../../components/Dropdown/Dropdown'
+import {DropdownThumbnail} from '../../components/DropdownThumbnail'
+import Header from '../../components/Header'
+import Footer from '../../components/Footer'
+
+
+export default function Contributors({ match }) {
+    const affiliation = match.params.affiliation
+
+    const classes = useStyle();
+
+    const [orgs, setOrgs] = useState([]);
+    const [affiliatedOrgs, setAffiliatedOrgs] = useState([]);
+    const [unAffiliatedOrgs, setUnAffiliatedOrgs] = useState([]);
+    const [affiliatedOpen, setAffiliatedOpen] = useState(false);
+    const [unaffiliatedOpen, setUnaffiliatedOpen] = useState(false);
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const result = await axios.get('https://test-civictechindexadmin.herokuapp.com/api/organizations/');
```

Comment:
Pull the Base URI as an environment variable

process.env.REACT_APP_BASE_URL + "/api/organizations"